multi_livechat/models/mail_channel.py

```diff
@@ -21,16 +26,17 @@ def _prepare_multi_livechat_channel_vals(
     ):
         return {
             "channel_partner_ids": [(4, pid) for pid in partner_ids],
-            "public": "groups",
-            "group_public_id": self.env.ref("base.group_user").id,
+            # "public": "groups", # V16 dropout public field
+            "group_public_id": False,  # V16 checks contrains with group_public_id self.env.ref("base.group_user").id,
             "channel_type": channel_type,
             "name": channel_name,
         }
 
     def _compute_is_pinned(self):
         # TODO: make batch search via read_group
         for r in self:
-            r.is_pinned = self.env["mail.channel.partner"].search_count(
+            # V16 change mail.channel.partner to mail.channel.member
+            r.is_pinned = self.env["mail.channel.member"].search_count(
```

Comment:
I agree with the change, but I don't think we need to add this comment. It makes sense in the commit description, but not in the code

Reply:
Ok, I moved out the comments.